app/src/main/java/com/ftw/hometerview/ui/login/LoginActivity.kt

```diff
@@ -69,25 +80,49 @@ class LoginActivity : AppCompatActivity() {
         binding.dotsIndicator.setViewPager2(binding.guideViewpager)
     }
 
+    private fun ObserveStartActivity(){
```

Comment:
```suggestion
    private fun observeStartActivity(){
```
+ `observe` 함수명이 전체적으로 통일되면 좋겠다! 한번 이야기해 보쟈!

Reply:
아하! 알겠습니다~